src/frontend/src/pages/Auth/Login.tsx

```diff
@@ -0,0 +1,146 @@
+import { Modal, ModalOverlay } from "components/base/Modal/Modal";
+import InputText from "components/InputText";
+import { login } from "api/auth";
+import EchoTextLogo from "assets/icons/echo-logo-text.svg?react";
+import NeighborhoodImage from "assets/icons/neighborhood.svg?react";
+import MailImage from "assets/icons/mail.svg?react";
+import loginInStyles from "./Login.styles";
+import type { RootState } from "store/store";
+
+const Login = () => {
+    const token = useSelector((state: RootState) => state.auth.token);
+    const [isOpen, setIsOpen] = useState(false);
+    const [email, setEmail] = useState<string>("");
+    const [isEmailSent, setIsEmailSent] = useState<boolean>(false);
+    const [hasError, setHasError] = useState<boolean>(false);
+
+    const { lang } = useParams<{ lang: LanguageKey | undefined }>();
+    const isDesktop = useMediaQuery("(min-width: 768px)");
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+
+    const {
+        root,
+        content,
+        captionContainer,
+        caption,
+        modalContainer,
+        modalTitle,
+        mailImageContainer,
+        modalErrorText,
+    } = loginInStyles({
+        isMobile: !isDesktop,
+    });
+
+    useEffect(() => {
+        if (!!token) navigate(`/${lang}/discover`, { replace: true });
+    }, [token]);
+
+    const onClickLogin = async () => {
+        try {
+            setHasError(false);
+            await login(email);
+            setIsEmailSent(true);
+        } catch (err) {
+            console.error(err);
+            setHasError(true);
+        }
+    };
+
+    const onChangeEmail = (value: string) => {
+        setHasError(false);
+        setEmail(value);
+    };
+
+    return (
+        <div className={root()}>
+            <SelectLanguageButtons language={lang || Language.EN} />
+            <div className={content()}>
+                {!isDesktop && <EchoTextLogo className="h-[32.4px]" />}
+                <NeighborhoodImage />
+                <div className={captionContainer()}>
+                    <p className={caption()}>{t("signInPageCaption")}</p>
+                </div>
+            </div>
+            <Button
+                variant="primary"
+                size="large"
+                className={isDesktop ? "" : "w-full"}
+                onPress={() => setIsOpen(true)}
+            >
+                {t("signIn")}
+            </Button>
+            <ModalOverlay
+                isDismissable
+                isOpen={isOpen}
+                onOpenChange={setIsOpen}
+            >
+                <Modal>
+                    <Dialog aria-label="Sign in with email">
+                        <div className={modalContainer()}>
+                            {!isEmailSent ? (
+                                <>
+                                    <p className={modalTitle()}>
+                                        {t("signInWithEmail")}
+                                    </p>
```

Comment:
There's a [subtitle in the figma designs](https://www.figma.com/design/LAy4vzpOLf7YnXbB7pGfN8/BHA-ECHO?node-id=754-16791&m=dev) to include here with an inline link to[ bha echo site](https://www.bostonhousing.org/en/Section-8-Leased-Housing/Voucher-Programs/Expanding-Choice-in-Housing-Opportunities-(ECHO).aspx)

Reply:
Good call. I thought we would drop this subtitle due to previous auth related discussions. Added it in https://github.com/azavea/echo-locator/pull/723/commits/1cb56972498f3b54df9692a493c7c9fb4f669c1d

<img width="423" height="722" alt="Screenshot 2025-10-06 at 2 14 58 PM" src="https://github.com/user-attachments/assets/84038e66-edbe-44fe-b2a3-85ab753bb99b" />
<img width="400" height="708" alt="Screenshot 2025-10-06 at 2 15 20 PM" src="https://github.com/user-attachments/assets/2039cbae-c647-4ba9-ac0b-5105c1d2724d" />
<img width="395" height="686" alt="Screenshot 2025-10-06 at 2 26 51 PM" src="https://github.com/user-attachments/assets/f2a8bf3c-7ec6-4a95-bddb-f9fd77fc65b7" />